compose-builder/add-asc-http-export-secure.yml

```diff
@@ -26,25 +26,25 @@ services:
     command: "/app-service-configurable ${DEFAULT_EDGEX_RUN_CMD_PARMS}"
     ports:
       - 127.0.0.1:48102:48102/tcp
-    container_name: app-service-http-export-secrets
-    hostname: app-service-http-export-secrets
+    container_name: app-http-export-secrets
```

Comment:
I was considering this can, but need to make sure this doesn't impact nay TAF tests.

Reply:
Also not sure why we need to have non-secure and separate secure instance of ASC running in TAF secure compose file. With secure Consul they both need to be secure... I think we need to rethink this and just have `add-asc-http-export-secure.yml` extend `add-asc-http-export.yml` to add security stuff, rather than be a separate instance... @cherrycl , @cloudxxx8 thoughts?

Reply:
> IMO, yes and address any TAF tests that depend on it.

Ok, maybe not. We don't do this with the core/support services... I.e. `core data` is simply `data`, but we do have the `hostname` & `container` name match the service key as you have done here.

Reply:
> Also not sure why we need to have non-secure and separate secure instance of ASC running in TAF secure compose file. With secure Consul they both need to be secure... I think we need to rethink this and just have `add-asc-http-export-secure.yml` extend `add-asc-http-export.yml` to add security stuff, rather than be a separate instance... @cherrycl , @cloudxxx8 thoughts?

Yes, I agree with @lenny-intel 's suggestion.  It could make the test cleaner.  I saw the PR has addressed this concern.  Thanks!

Reply:
Oh, this is not addressed in taf folder.  I will discuss with @cherrycl and open another PR to address this.
Let's discuss in https://github.com/edgexfoundry/edgex-compose/issues/59

Reply:
updated the compose files to reflect this.  the -secure one now extends from the base version.